src/main/resources/init_db.sql

```diff
@@ -0,0 +1,10 @@
+CREATE DATABASE db;
+
+USE db;
```

Comment:
you shouldn't create whole database in init script

you should rather create a schema (optional) and tables that you need